wikipron/config.py

```diff
@@ -135,36 +138,24 @@ def wrapper(pron):
 
         return wrapper
 
-    def _get_li_selector(self, language: str, dialect: Optional[str]) -> str:
+    def _get_pron_xpath_selector(
+        self, language: str, dialect: Optional[str]
+    ) -> str:
         if not dialect:
             dialect_selector = ""
         else:
-            dialect_selector = _DIALECT_SELECTOR_TEMPLATE.format(
+            dialect_selector = _DIALECT_XPATH_SELECTOR_TEMPLATE.format(
                 dialects_text=" or ".join(
                     f'text() = "{d.strip()}"' for d in dialect.split("|")
                 )
             )
 
-        return _LI_SELECTOR_TEMPLATE.format(
+        return _PRON_XPATH_SELECTOR_TEMPLATE.format(
             language=language, dialect_selector=dialect_selector
         )
 
-    def _get_process_word(
-        self, cut_off_date: Optional[str]
-    ) -> Callable[[str, str], str]:
-        def wrapper(word, date):
-            # Skips multiword examples.
-            if " " in word:
-                return None
-            # Skips examples containing a dash.
-            if "-" in word:
-                return None
-            # Skips examples containing digits.
-            if re.search(r"\d", word):
-                return None
-            # Skips examples available later than the cut-off date.
-            if date > cut_off_date:
-                return None
-            return word
-
-        return wrapper
```

Comment:
This bit is now separated between the `_skip_word` and `_skip_date` functions, which I now strongly prefer to my previous convoluted version here that tied word and date together.

Reply:
Nice, this is a lot clearer, yes.